tslint.json

```diff
@@ -7,6 +7,5 @@
   },
   "rulesDirectory": [
     "node_modules/tslint-eslint-rules/dist/rules"
-  ],
-  "extends": "tslint-ionic-rules"
+  ]
```

Comment:
Nope, it is also a part of the update. things are constantly being changed by ionic so it might be added again. I am just going with the framework as it goes. It's not stable as I thought!